UIDD/UIDD-Session8/consecutiveString.js

```diff
@@ -0,0 +1,19 @@
+//Remove Consecutive Repeating String
```

Comment:
Issue observed in provided input "eeabcddcbfgf".

Reply:
The output of this string given in Question is wrong.
Output will be "afgf"